src/main/java/com/lpvs/util/LPVSPayloadUtil.java

```diff
@@ -24,7 +28,29 @@
  * specific details such as organization name, repository name, repository URL, and pull request ID.
  */
 @Slf4j
-public class LPVSWebhookUtil {
+public class LPVSPayloadUtil {
+
+    /**
+     * Creates an InputStreamReader for the given process, handling UTF-8 encoding.
+     *
+     * @param inputStream The input stream to read from.
+     * @return An InputStreamReader for the error stream of the process.
+     * @throws UnsupportedEncodingException If UTF-8 encoding is not supported.
+     */
+    public static InputStreamReader createInputStreamReader(InputStream inputStream)
+            throws UnsupportedEncodingException {
+        return new InputStreamReader(inputStream, "UTF-8");
```

Comment:
Is returning the `error` stream correct? 
I think either description or code implementation should be modified.

Reply:
Thanks. The description is outdated.